book/P1C6_survtsk.qmd

```diff
@@ -10,119 +10,270 @@ abstract: "This chapter introduces the different survival problems and formalise
-3. Aiding in interpretability -- in some cases this could simply be $g(\bseta) = -\bseta$ to ensure the 'higher value implies higher risk' interpretation.
+1. Scaling or normalization: simple functions to scale the linear predictor can better support interpretation and visualisation;
+2. Capturing non-linear effects: for example the Cox PH model (@sec-models-classical) applies the transformation $g(\bseta) = \exp(\bseta)$ to capture more complex relationships between features and outcomes;
+3. Aiding in interpretability: in some cases this could simply be $g(\bseta) = -\bseta$ to ensure the 'higher value implies higher risk' interpretation.
 
-A prognostic index is a special case of the survival ranking task, assuming that there is a one-to-one mapping between the prediction and expected survival times.
-Once again, it is assumed in this book that a higher value for the prognostic index implies higher risk of event.
+### Prognostic index, risks, and times
 
-<!-- FIXME example needed -->
+A prognostic index is a special case of the survival ranking task, assuming that there is a one-to-one mapping between the prediction and expected survival times, i.e., as long as there is an intuitive relationship that explains how the prognostic index maps to survival time. 
+For example, one could define $g(\bseta) = 1 \text{ if } \eta < \text{5 or } 0 \text{ otherwise }$ however this would prevent the prognostic index being used as a ranking method and it would defeat the point of a prognostic index.
 
-## Predicting Distributions {#sec-survtsk-dist}
+As with a risk prediction, there is no general relationship between the prognostic index and survival time.
+However, there are cases where there may be a clear relationship between the two.
+For example, one can theoretically estimate a survival time from the linear predictor used in an accelerated failure time model (@sec-surv-models).
+<!-- FIXME: UPDATE REFERENCE ABOVE WITH UPDATED VERSION WHEN MERGED -->
+
+### Prognostic index and distributions
+
+A general prognostic index cannot be generated from a distribution, however the reverse transformation is very common in survival models.
+The vast majority of survival models are composed from a group-wise survival time estimate and an estimation of an individual's prognostic index.
+This will be seen in detail in future chapters (particularly in @sec-boost and @sec-nn) but in general it is very common for models to assume data follows either proportional hazards or accelerated failure time (@sec-models-classical).
+In this case models are formed of two components:
+
+1. A non-parametric estimator (@sec-surv-km) that estimates the baseline hazard, $\hatbh$, or baseline survival function, $\hatbS$. Discussed further in @sec-models-classical, these are group-wise estimates that apply to the entire sample.
```

Comment:
I think this is misleading. In the AFT, one can write down the Survival probability in terms of a baseline Surv prob S_0(t) multiplied or exponantiated with individual risk, but that doesn't make it non-parameteric. The model itself is still fully parameteric and the parameters that govern S_0(t) are estimated/calculated parametrically.

Reply:
That also usually breaks down, when the shape parameter of the model can depend on covariates.